KomaMRICore/src/sequences/PulseDesigner.jl

```diff
@@ -4,8 +4,8 @@
 A module to define different pulse sequences.
 """
 module PulseDesigner
-using ..KomaMRI
-using ..KomaMRI: γ, Scanner, get_bvalue, get_max_grad
+using ..KomaMRICore
+using ..KomaMRICore: γ, Scanner, get_bvalue, get_max_grad
```

Comment:
Is this `using ..KomaMRICore: γ, Scanner, get_bvalue, get_max_grad` necessary?

Reply:
Apparently Scanner is necessary. γ, get_bvalue and get_max_grad will be removed.

Reply:
The problem is that we export `Sequence`, `Scanner`, `Grad`, `RF`, `ADC`, `Delay`, `γ`, etc **after** we define the submodule with `include("sequences/PulseDesigner.jl")`. If we put the definition of PulseDesigner in the end with the definition of the UI, it works correctly:

```julia
#Main
export γ #gyro-magnetic ratio [Hz/T]
export Scanner, Sequence, Phantom
export Grad, RF, ADC, Delay, dur

...

#PulseDesigner submodule
include("sequences/PulseDesigner.jl")
export PulseDesigner
#GUI
include("KomaUI.jl")
export KomaUI

...

```